src/broadcast/types.ts

```diff
@@ -14,3 +16,46 @@ export interface StartBroadcastConfig {
   authToken: string;
   name?: string;
 }
+
+export enum BroadcastEvent {
+  SLIPPI_STATUS_CHANGE = "slippiStatusChange",
+  DOLPHIN_STATUS_CHANGE = "dolphinStatusChange",
+  ERROR = "broadcastError",
+  LOG = "log",
```

Comment:
```suggestion
  SLIPPI_STATUS_CHANGE = "SLIPPI_STATUS_CHANGE",
  DOLPHIN_STATUS_CHANGE = "DOLPHIN_STATUS_CHANGE",
  ERROR = "ERROR",
  LOG = "LOG",
```

While we're refactoring these, can we make it like this?